app/views/search/_sitetour_modal.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="modal fade sitetour" id="sitetourModal" style="font-family: FontAwesome;" tabindex="-1" role="dialog">
```

Comment:
Because I think FontAwesome is pretty and it was a one-timer.  I'll remove it ;)